### PR TITLE
Extend on the redjudging overview

### PR DESCRIPTION
**Pull request by mvr320**
Added the individual judging runs to it, still to be done is the testcase output

image

### CODE REVIEW

**Comment by meisterT** [comment]
Can you please add a screenshot?

File: webapp/templates/jury/rejudging.html.twig [hunk 1]
```diff
@@ -148,6 +148,20 @@
...
+            <h4>Judging verdicts</h4>
+            <table class="table table-sm table-striped">
+                <tr><th>jID</th><th>rID</th><th>Judgehost</th><th>runtime</th><th>verdict</th></tr>
+                {% for judging in stats.judgings %}
+                    <tr>
+                        <td>{{ judging.judgingid }}</td>
+                        <td>{{ judging.rejudgingid }}</td>
+                        <td>{{ judging.hostname }}</td>
+                        <td>{{ judging.duration }}</td>
+                        <td>{{ judging.result }}</td>
```
Comment (meisterT) [inline webapp/templates/jury/rejudging.html.twig:161]: please add a ` | printResult` here

File: webapp/templates/jury/rejudging.html.twig [hunk 2]
```diff
@@ -148,6 +148,20 @@
...
+
+            <h4>Judging verdicts</h4>
+            <table class="table table-sm table-striped">
+                <tr><th>jID</th><th>rID</th><th>Judgehost</th><th>runtime</th><th>verdict</th></tr>
+                {% for judging in stats.judgings %}
+                    <tr>
+                        <td>{{ judging.judgingid }}</td>
+                        <td>{{ judging.rejudgingid }}</td>
+                        <td>{{ judging.hostname }}</td>
+                        <td>{{ judging.duration }}</td>
```
Comment (meisterT) [inline webapp/templates/jury/rejudging.html.twig:160]: please add a ` | number_format(3)` here

**Review by meisterT — APPROVED** [review]
I think this already worth merging after you fixed the two nits below.

File: webapp/templates/jury/rejudging.html.twig (hunk 1 above)
Comment (mvr320) [reply webapp/templates/jury/rejudging.html.twig:161]: What will this do?

Comment (nickygerritsen) [reply webapp/templates/jury/rejudging.html.twig:161]: Make it red / green etc

**Comment by meisterT** [comment]
🚢 it 😉

**Pull request merged**